Build/build-functions.psm1

```diff
@@ -58,7 +58,8 @@ function Start-Build([boolean] $skipUWP = $false) {
 function Start-Tests {
   $projectPaths = @(
     "UnitsNet.Tests\UnitsNet.Tests.NetCore.csproj",
-    "UnitsNet.Serialization.JsonNet.Tests\UnitsNet.Serialization.JsonNet.Tests.NetCore.csproj"
+    "UnitsNet.Serialization.JsonNet.Tests\UnitsNet.Serialization.JsonNet.Tests.NetCore.csproj",
+    "UnitsNet.Serialization.JsonNet.CompatibilityTests\UnitsNet.Serialization.JsonNet.CompatibilityTests.NetCore.csproj"
```

Comment:
Oh god I love github and that I can fix minor things like whitespace issues directly in the browser instead of nagging about it :heart:
It was just slightly misaligned, possibly a tab char gone wrong. Now fixed.